Shared/TLS/Get-TlsCipherSuiteInformation.ps1

```diff
@@ -0,0 +1,62 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+. $PSScriptRoot\..\Get-RemoteRegistryValue.ps1
+function Get-TlsCipherSuiteInformation {
+    [OutputType("System.Object")]
+    param(
+        [string]$MachineName = $env:COMPUTERNAME,
+        [scriptblock]$CatchActionFunction
+    )
+
+    begin {
+        Write-Verbose "Calling: $($MyInvocation.MyCommand)"
+    }
+    process {
+        try {
+            # 'Get-TlsCipherSuite' takes account of the cipher suites which are configured by the help of GPO.
+            # No need to query the ciphers defined via GPO if this call is successful.
+            Write-Verbose "Trying to query TlsCipherSuites via 'Get-TlsCipherSuite'"
+            $tlsCipherSuites = Get-TlsCipherSuite -ErrorAction Stop
+        } catch {
```

Comment:
This isn't getting the information remotely. This needs to be addressed.

Reply:
Oops, you're right. Good catch.

Reply:
This call is now performed via `Invoke-ScriptBlockHandler`